_posts/blog/2015-11-08-gtac.md

```diff
@@ -15,13 +15,87 @@ date: 2015-11-08T15:39:55-04:00
+<a href="https://docs.google.com/presentation/d/1DxEX7d3UZKFv-ekHzt1-eRCWtlR8uAyXt6ra1B14IGQ/pub">slides</a>
+
+Fanzy robot hands and fingers combined with a high speed camera that allow reproducible tests on every touchscreen. There main purpose is to test the accuracy and speed of displays and the devices in general. But there are so cool, I would love to have one of these things at work.
+
+### Chromecast Test Automation
+<iframe width="560" height="315" src="https://www.youtube.com/embed/TfAVJLhb_k0?list=PLSIUOFhnxEiCWGsN9t5A-XOhRbmz54IS1" frameborder="0" allowfullscreen></iframe>
+<a href="https://docs.google.com/presentation/d/1gBGS5UFyyR0mcwg990I1YDOFQwAhL-nmonNPGvGYnlk/pub">slides</a>
+
+Did you ever asked yourself how Google tests hardware like the chromecasts? Well this talk gives you the answer. In a Google way - means in a massive scale with a lot of intelligence behind it.
+
+### Your Tests Aren't Flaky
+<iframe width="560" height="315" src="https://www.youtube.com/embed/hmk1h40shaE?list=PLSIUOFhnxEiCWGsN9t5A-XOhRbmz54IS1" frameborder="0" allowfullscreen></iframe>
+<a href="https://docs.google.com/presentation/d/1L9hGYqCAgjZyXE9ch4Toh4ziuYYkB2OiMCdFpgfTko0/pub?pageId=109506508449676915259&slide=id.gd8d3f5279_0_0">slides</a>
+
+I'm pretty sure you know sentences like this: "Ahh - your test is just flaky, just re-run it. If it is green at the second run, everything is fine!" Well this talk from Alister Scott tells you a different story. Something like flaky tests doesn't exist - if your test results are non-deterministic, it could actually be the fault of the system under test. If you want more after watching his tals - he has a highly recommandable <a href="http://watirmelon.com/">blog (watirmelon.com)</a> that gives a lot tips about test automation on general.
+
+### Large-Scale Automated Visual Testing
+<iframe width="560" height="315" src="https://www.youtube.com/embed/euJ2OrlBEqQ" frameborder="0" allowfullscreen></iframe>
+<a href="https://docs.google.com/presentation/d/1TslcNggA6xx_GQbYu4QYU9vozdHhZ2pQE1Kj52ch1VQ/pub">slides</a>
+
+The grande final from Day 1 was presented by Adam Carmi from <a href="https://applitools.com/">Applitools</a>. He showed us Applitools Eyes - a framework for visual testing with massive intelligence in the background. There is a integration into nearly all common used web and mobile testing frameworks to upload screenshots to Applitools. Once uploaded - one of the screens are declared as a "golden master" and all succesive uploads of this test step are compared with this master. Maybe you aren't impressed now - but actually you should, because this isn't just a simple pixel perfect comparison. It allows e.g. to compare just the structure of the mobile/web app without consideration of the content. Also the resolution doesn't matter - so it is possible to compare screens from a tablet with one from a phone.
```

Comment:
The grand final of day 1
There is an integration .... commonly used ... one of the screens is declared
Maybe you're not impressed yet - but ....
It allows you to compare the structure of any mobile/web...